driver/src/main.rs

```diff
@@ -154,10 +154,15 @@ struct Options {
     )]
     solver_time_limit: Duration,
 
-    /// Solver parameter: minimal average fee per order
-    /// Its unit is [OWL]
+    /// Subsidy factor used to compute the minimum average fee per order in a
+    /// solution.
+    #[structopt(long, env = "MIN_AVG_FEE_SUBSIDY_FACTOR", default_value = "10.0")]
```

Comment:
Could this not be the same as the other subsidy factor as they are directly related? I'm worried that if they are too far apart we might get a bunch of solutions which eventually don't pass as we are not willing to retry with a high enough gas price.

Reply:
This was also unclear to me since these factors have slightly different semantics. I'd be happy with using just one as well though.

Reply:
Do you have any ideas how we can merge these two subsidy factors into one? Maybe they can be a multiple of each other?

Reply:
Can't they be the same factor?
Here we say that the solution should earn at least as much ether as it costs to submit it at fast gas price. This value is then divided by subsidy factor.
In the other place we say that the gas price cap is the amount earned through fees. This value is then multiplied by subsidy factor.
Either way we use the subsidy factor to effectively divide the gas price.

Reply:
Sounds reasonable to me.

Reply:
Agree it should be the same. Maybe we could even move the logic to have `ApproximateMinAverageFee` also be able to tell you the gas price given some solution.